assets/blocks/shared.js

```diff
@@ -1,20 +1,23 @@
 /**
  * WordPress dependencies
  */
-import { unregisterBlockType } from '@wordpress/blocks';
+import { unregisterBlockType, getBlockTypes } from '@wordpress/blocks';
 import { subscribe, select } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import registerSenseiBlocks from './register-sensei-blocks';
 import ContactTeacherBlock from './contact-teacher-block';
+import LearnerCoursesBlock from './learner-courses-block';
 
+// Post types where blocks should be loaded. Or null if it should be loaded for any post type.
 const BLOCKS_PER_POST_TYPE = {
 	'sensei-lms/button-contact-teacher': [ 'course', 'lesson' ],
+	'sensei-lms/learner-courses': null,
```

Comment:
We could just don't add it here and it wouldn't be removed from any post type, but I think it's nice to have here for documentation purpose, and maybe for future purposes.

Reply:
That would also mean re-thinking bundling it with shared styles/JS, and the naming of `Sensei_Global_Blocks`. Maybe we can refer to their function for the grouping, though not sure what's the best word. Could be Site blocks or Template blocks?

Reply:
Refactored here: https://github.com/Automattic/sensei/pull/4122/commits/daf9c056d0123c27d930bfa9d43e79013b36f108